### ISSUE TITLE
Tuya PH meter works with HA?

### ISSUE THREAD

**Issue by matata86** [rospogrigio/localtuya#1088]
Hello, can i ask you... Works this product with your addon and is possible send data to HA?
https://www.aliexpress.com/item/1005003778452874.html?spm=a2g0o.productlist.0.0.12c26a2d8Nw5qe&algo_pvid=edc7364d-9b8b-4dd3-a184-01d2760882b8&algo_exp_id=edc7364d-9b8b-4dd3-a184-01d2760882b8-0&pdp_ext_f=%7B%22sku_id%22%3A%2212000027137298761%22%7D&pdp_npi=2%40dis%21CZK%212355.45%211790.03%21%21%21%21%21%400b0a0ac216659943601418710efcf1%2112000027137298761%21sea&curPageLogUid=85DP8yBqOvoY

Thank you.

**Comment by skidude89** [comment]
Yes, I just setup the exact same device. While it didn't work with the regular Tuya integration, using localtuya I was able to get the sensors setup in Home Assistant

**Comment by remlei** [comment]
You can just add the sensor DPs one by one as entity in Home Assistant, it should work on every single device out there, as long as its not battery powered. 

Its just a matter if your competent enough to identify the DPs corresponding to the sensors. For experienced user, they should be able to identify each DP of the device by just comparing the values from Smartphone and from the iot developer website of tuya.

For example I managed to make a mmwave wifi based sensor work in localtuya by just manually adding the entity one by one to home assistant, the hardpart really is identfying the DP which took about 15minutes of my time comparing stuff.

**Comment by MoonshineSG** [comment]
Hi... can you detail "add the sensor DPs one by one as entity in Home Assistant," ?
